game.py

```diff
@@ -405,26 +478,109 @@ def _button_press_cb(self, win, event):
         return True
-
+        
+    def _game_over(self):
+        for opt in self._opts:
+            opt.hide()
+            opt.type = -1
+        for dot in self._dots:
+            dot.hide()
+        for question_shape in self._question:
+            question_shape.hide()
+        self.save_highscore()
+        yoffset = int(self._space / 4.)
+        xoffset = int((self._width - 6 * self._dot_size - \
+                                   5 * self._space) / 2.)
+        y = 1
+        i=0
+        for x in range(2,6):
+            self._gameover.append(
+                Sprite(self._sprites,
+                    xoffset + (x-0.25) * (self._dot_size - 15),
+                    y * (self._dot_size - 90+ self._space) + yoffset,
+                    self._new_dot_surface(color=self._colors[1])))
+            self._gameover[-1].type = -1  # No image
+            self._gameover[-1].set_label_attributes(72)
+            i+=1
+        text = []
+        text.append("☻")
+        text.append("  Game  ")
+        text.append("  Over  ")
+        text.append("☻")
+        i = 0
+        for x in range(4) :
+            self._gameover[x].type = -1
+            self._gameover[x].set_shape(self._new_dot_surface(
+                        self._colors[2]))
+            self._gameover[x].set_label(text[i])
+            self._gameover[x].set_layer(100)
+            i += 1
+        y = 2
+        for x in range(2,5):
+            self._score.append(
+                Sprite(self._sprites,
+                    xoffset + (x + 0.25) * (self._dot_size - 15),
+                    y * (self._dot_size -30 + self._space) + yoffset,
+                    self._new_dot_surface(color=self._colors[1])))
+            self._score[-1].type = -1  # No image
+            self._score[-1].set_label_attributes(72)
+        text = []
+        text.append("  your  ")
+        text.append(" score:  ")
+        text.append("  " + str(self._correct) + "  ")
+        i = 0
+        for x in range(3):
+            self._score[x].type = -1
+            self._score[x].set_shape(self._new_dot_surface(
+                        self._colors[2]))
+            self._score[x].set_label(text[i])
+            self._score[x].set_layer(100)
+            i += 1
+        y = 3   
+        if (self._correct > self._high_score_count):
+            self._high_score_count = self._correct
+        for x in range(2,5):
+            self._highscore.append(
+                Sprite(self._sprites,
+                    xoffset + (x + 0.25) * (self._dot_size - 15),
+                    y * (self._dot_size -20+ self._space) + yoffset,
+                    self._new_dot_surface(color=self._colors[1])))
+            self._highscore[-1].type = -1  # No image
+            self._highscore[-1].set_label_attributes(72)
+        text = []
+        text.append("  high  ")
+        text.append(" score:  ")
+        text.append("  " + str(self._high_score_count) + "  ")
```

Comment:
You might like to use string formatting here to make it look neater

```suggestion
        text = []
        text.append("  high  ")
        text.append(" score:  ")
        text.append(f"  {self._high_score_count}  ")
```